app/models/lookup_value.rb

```diff
@@ -34,7 +34,7 @@ def check_permissions_after_save
 
   def value=(val)
     if val.is_a?(HashWithIndifferentAccess)
-      super(val.deep_to_hash)
+      super(val.to_hash)
```

Comment:
This should use `deep_stringify_keys`, and perhaps do it for all hashes, not just HWIA's (in case there is a HWIA nested inside a regular hash)